package.json

```diff
@@ -35,8 +35,9 @@
     "cookie-parser": "^1.4.1",
     "cordova": "^6.1.1",
     "express": "^4.13.4",
-    "express-session": "^1.13.0",
     "ionic": "^1.7.14",
+    "spotify-web-api-node": "^2.3.0",
+    "express-session": "^1.13.0",
     "passport": "^0.3.2",
     "passport-spotify": "^0.3.0"
   }
```

Comment:
please move body-parser (ln23, won't let me comment there) to the dependencies from the dev ones